assets/js/components/adminbar/AdminBarClicks.js

```diff
@@ -88,27 +84,14 @@ const AdminBarClicks = ( { className } ) => {
 	const totalClicksChange = calculateChange( totalOlderClicks, totalClicks );
 
 	return (
-		<div className={ classnames(
-			'mdc-layout-grid__cell',
-			className,
-		) }>
-			<DataBlock
-				className="overview-total-clicks"
-				title={ __( 'Total Clicks', 'google-site-kit' ) }
-				datapoint={ totalClicks }
-				change={ totalClicksChange }
-				changeDataUnit="%"
-			/>
-		</div>
+		<DataBlock
+			className="overview-total-clicks"
+			title={ __( 'Total Clicks', 'google-site-kit' ) }
+			datapoint={ totalClicks }
+			change={ totalClicksChange }
+			changeDataUnit="%"
+		/>
 	);
-};
-
-AdminBarClicks.propTypes = {
-	className: PropTypes.string,
-};
-
-AdminBarClicks.defaultProps = {
-	className: 'mdc-layout-grid__cell--span-2-tablet mdc-layout-grid__cell--span-3-desktop',
-};
+}
 
-export default AdminBarClicks;
+export default withWidgetComponentProps( { widgetSlug: WIDGET_SLUG } )( AdminBarClicks );
```

Comment:
I think it's a bit unintuitive passing this here. The outer `AdminBarWidgets` component right now has the same widget slugs from all the sub-components hard-coded, and the connection between the `WIDGET_SLUG` constants from these sub-components and the same values being used in `AdminBarWidgets` is not clear.

We should think of these sub-components as if they were components for registered widgets. If that was the case, they would receive all widget component props and they don't need to define their own slugs - that part would happen at widget registration.

I think it would be better to make all the individual sub-component files fully agnostic of their "slugs", the same way that a registered widget's component does not need to be aware of the slug. In other words:
* Let's _only_ declare these slugs within the `AdminBarWidgets` file.
* From that component, let's wrap the four sub-components individually with `withWidgetComponentProps`, each one for the given slug.
* The sub-component files should remain agnostic and simply return the component, which should expect to receive the typical widget component props.